tar/bsdtar.c

```diff
@@ -1008,11 +1017,6 @@ main(int argc, char **argv)
 		break;
 	}
 
-	cleanup_exclusions(bsdtar);
```

Comment:
These are unchanged from libarchive, aren't they?  In general I think "keep code consistent with libarchive" trumps "reorganize code to make it cleaner".